File: MatDiscreta/src/Negocios/Variable.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Negocios;

/**
 *
 * @author Gabriel
 */
public class Variable {
    private boolean Valor;
        public Variable(boolean Negado, String Valor)
        {
            boolean valor = (Valor == "V") ? true : false;
            if (Negado)
                this.Valor = !valor;
            else
                this.Valor = valor;
        }
        public Variable(boolean Valor)
        {
            this.Valor = Valor;
        }
        public boolean Retorno()
        {
            return Valor;
        }
}

File: Seccion 1/Alternativa para Programadores/Seccion 1.3/Codigo/Variable.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace TautologiaContingenciaContradiccion
{
    public class Variable
    {
        private bool Valor;
        public Variable(bool Negado, string Valor)
        {
            bool valor = (Valor == "V") ? true : false;
            if (Negado)
                this.Valor = !valor;
            else
                this.Valor = valor;
        }
        public Variable(bool Valor)
        {
            this.Valor = Valor;
        }
        public bool Retorno()
        {
            return Valor;
        }
    }
}
